core/embed/rust/src/ui/flow/swipe.rs

```diff
@@ -296,23 +322,22 @@ impl SwipeFlow {
 /// This implementation relies on the fact that swipe components always return
 /// `FlowMsg` as their `Component::Msg` (provided by `impl FlowComponentTrait`
 /// earlier in this file).
-#[cfg(feature = "micropython")]
-impl ObjComponent for SwipeFlow {
-    fn obj_place(&mut self, bounds: Rect) -> Rect {
+impl Layout<Result<Obj, Error>> for SwipeFlow {
```

Comment:
Commentary above this `impl` is now outdated.

Reply:
333c5ec